plugins/janus_videoroom.c

```diff
@@ -1488,6 +1488,28 @@ typedef enum  forward_media_type {
 				(MEDIA_VIDEO   == media_type ? "Video"  :              \
 				(MEDIA_AUDIO_MIXER == media_type ? "AudioMixer" : "unknown"))))
 
+#define RETURN_FALSE_IF_EQUAL_LOG_ERROR(value1, value2, format, ...) \
```

Comment:
VERIFY_ELSE_RETURN_FALSE

in this style

#define VERIFY_ELSE_CLEANUP(condition, message)          \
	if(!(condition))                                     \
	{                                                    \
		GST_ERROR((message));                            \
		goto CLEANUP;                                    \
	}

Reply:
done